elasticsearch/prep-install.origin

```diff
@@ -7,9 +7,13 @@ if [ -z "${ES_CLOUD_K8S_URL:-}" ] ; then
     ES_CLOUD_K8S_URL=io.fabric8/elasticsearch-cloud-kubernetes/${ES_CLOUD_K8S_VER}
 fi
 if [ -z "${OSE_ES_URL:-}" ] ; then
-    OSE_ES_URL=io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+    MANPATH="" source /opt/rh/rh-maven33/enable 
+    pushd /tmp/lib/openshift-elasticsearch-plugin
+      mvn clean verify -DskipTests
+      OSE_ES_URL=file:///$(pwd)/target/releases/openshift-elasticsearch-plugin-${OSE_ES_VER}.zip
```

Comment:
I think you have too many `/` in there - should be `file://$(pwd)/target...` ?  either that, or `file:////` is acceptable?